tests/testing.py

```diff
@@ -130,17 +135,21 @@ def script(self, *args, env_update=None):
         process.stderr.close()
         return result
 
-    def setup_livepatch(self, installed=None, enabled=None,
+    def setup_livepatch(self, installed=False, enabled=None,
                         livepatch_command=None):
         """Setup livepatch-related fakes."""
-        if installed is not None:
-            command = (
-                SNAP_LIVEPATCH_INSTALLED if installed
-                else SNAP_LIVEPATCH_NOT_INSTALLED)
-            self.make_fake_binary('snap', command=command)
+        command = (
+            SNAP_LIVEPATCH_INSTALLED if installed
+            else SNAP_LIVEPATCH_NOT_INSTALLED)
+        self.make_fake_binary('snap', command=command)
```

Comment:
Can we make the same simplification for the enabled=None logic? I don't remember why we had these default to None. Tests still pass, but am I trusting them too much? :)
```diff
--- a/tests/testing.py
+++ b/tests/testing.py
@@ -135,19 +135,18 @@ class UbuntuAdvantageTest(TestWithFixtures):
         process.stderr.close()
         return result
 
-    def setup_livepatch(self, installed=False, enabled=None,
+    def setup_livepatch(self, installed=False, enabled=False,
                         livepatch_command=None):
         """Setup livepatch-related fakes."""
         command = (
             SNAP_LIVEPATCH_INSTALLED if installed
             else SNAP_LIVEPATCH_NOT_INSTALLED)
         self.make_fake_binary('snap', command=command)
-        if enabled is not None:
-            if livepatch_command is not None:
-                command = livepatch_command
-            else:
-                command = LIVEPATCH_ENABLED if enabled else LIVEPATCH_DISABLED
-            self.make_fake_binary('canonical-livepatch', command=command)
+        if livepatch_command is not None:
+            command = livepatch_command
+        else:
+            command = LIVEPATCH_ENABLED if enabled else LIVEPATCH_DISABLED
+        self.make_fake_binary('canonical-livepatch', command=command)
 
     def setup_esm(self, enabled=False):
         """Setup the ESM repository."""
```

Reply:
IIRC the reason for `None` there is that no `canonical-livepatch` command is present. I think we do check if the binary is present, so maybe tests pass because we still default to disabled.